src-built-in/components/userPreferences/src/components/general/Toolbar.jsx

```diff
@@ -38,27 +70,33 @@ export default class Toolbar extends React.Component {
 				monitor: value
 			});
 		});
+		//Unless we want to add another config variable tracking dockable state we need to determine state
+		//using the set values.
+		FSBL.Clients.ConfigClient.getValue("finsemble.components.Toolbar.window.dockable", (err, value) => {
+			if (!value) { 
+				value = "Fixed";
+			} else if (JSON.stringify(value) === JSON.stringify(["top", "bottom"])) {
+				value = "Floating";
+			}
+			this.setState({
+				toolbarType: value
```

Comment:
Should an invalid value get handed in we will just apply it blindly. I believe what we *should* do is before the `setState` call do something like: `if (["Fixed", "Floating"].includes(value) === false) { value = "Fixed"; }`. This guards against any invalid state value making its way in from further progressing down the chain. Along with setting the value we'd log an error to the logger for developers to hopefully see.